src/main/scala/awscala/dynamodbv2/AttributeValue.scala

```diff
@@ -37,6 +38,8 @@ object AttributeValue {
 
   def apply(v: aws.model.AttributeValue): AttributeValue = new AttributeValue(
     s = Option(v.getS),
+    //Unfortunately the ugly check is necessary here to bypass implicit conversion
+    bl = if (v.getBOOL != null) Option(v.getBOOL) else None,

```

Comment:
It's funny, I had assumed the same but it seems this still raises an NPE.  Will dig deeper:

```
[info]   java.lang.NullPointerException:
[info]   at scala.Predef$.Boolean2boolean(Predef.scala:361)
[info]   at awscala.dynamodbv2.AttributeValue$.apply(AttributeValue.scala:41)
[info]   at awscala.dynamodbv2.Item$$anonfun$apply$1.apply(Item.scala:9)
[info]   at awscala.dynamodbv2.Item$$anonfun$apply$1.apply(Item.scala:9)
[info]   at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:245)
[info]   at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:245)
[info]   at scala.collection.mutable.ResizableArray$class.foreach(ResizableArray.scala:59)
[info]   at scala.collection.mutable.ArrayBuffer.foreach(ArrayBuffer.scala:48)
[info]   at scala.collection.TraversableLike$class.map(TraversableLike.scala:245)
[info]   at scala.collection.AbstractTraversable.map(Traversable.scala:104)
```